tests/tensor/random/test_basic.py

```diff
@@ -237,6 +263,40 @@ def test_exponential_samples():
     rv_numpy_tester(exponential, test_lambda, size=[2, 3])
 
 
+def test_weibull_samples():
+    def test_fn(alpha, beta=1, size=None):
+        return beta * np.random.weibull(alpha, size=size)
+
+    test_alpha = np.array(3, dtype=config.floatX)
+    test_beta = np.array(0.5, dtype=config.floatX)
+
+    rv_numpy_tester(weibull, test_alpha, test_fn=test_fn)
+    rv_numpy_tester(weibull, test_alpha, test_beta, test_fn=test_fn)
+    rv_numpy_tester(weibull, test_alpha, test_beta, size=[2, 3], test_fn=test_fn)
+
+
+def test_logistic_samples():
+
+    test_loc = np.array(2, dtype=config.floatX)
+    test_scale = np.array(0.5, dtype=config.floatX)
+
+    rv_numpy_tester(logistic)
+    rv_numpy_tester(logistic, test_loc)
+    rv_numpy_tester(logistic, test_loc, test_scale, size=[2, 3])
+
+
+def test_vonmises_samples():
+    def test_fn(mu=0, kappa=1, size=None):
```

Comment:
I added default parameters to the vonmises, which are not default in numpy (which has none as default). Does it make sense to do this, and if so, does it make sense to use the `test_fn` as I did here?

Reply:
Same for Triangular...

Reply:
If they're not default in NumPy we don't want to introduce them here.